src/test/java/com/segment/analytics/android/integrations/amplitude/AmplitudeTest.java

```diff
@@ -493,6 +499,12 @@ public void screenTrackAllPages() {
   }
 
   @Test
+  public void screenTrackAllPagesV2() throws JSONException {
```

Comment:
We should add a test to make sure it not only sends `name` as the properties, but also any custom properties you pass in.

Reply:
Will do.